packages/vite-node/src/cli.ts

```diff
@@ -16,33 +16,42 @@ cli
   .option('-r, --root <path>', 'Use specified root directory')
   .option('-c, --config <path>', 'Use specified config file')
   .option('-w, --watch', 'Restart on file changes, similar to "nodemon"')
+  .option('--script', 'Use vite-node as a script runner')
```

Comment:
Can we have shorthand here too? @sheremet-va what do you think?

Reply:
I avoided the shorthand because it would deny forwarding `-s` to the executed script which is very common. The more we can pass to the executed script the better?